symplyphysics/laws/dynamics/total_work_equals_change_in_kinetic_energy.py

```diff
@@ -56,85 +48,27 @@
 The time after the work has been done.
 
 Symbol:
-    t1
+    :code:`t1`
 
 Latex:
     :math:`t_1`
 """
 
-law = Eq(total_work, kinetic_energy(time_after) - kinetic_energy(time_before))
+law = Eq(work, kinetic_energy(time_after) - kinetic_energy(time_before))
 r"""
-W = K(t1) - K(t0)
+:code:`W = K(t1) - K(t0)`
 
 Latex:
     .. math::
         W = K(t_1) - K(t_0)
 """
 
-# Derive the law in case of rectilinear motion with infinitesimal constant total force acting on particle and
+# TODO Derive the law in case of rectilinear motion with infinitesimal constant total force acting on particle and
```

Comment:
What's wrong with current proof?

Reply:
There was something wrong with an equation in the proof and I tried to make sense of what's the problem, but I couldn't understand what and the proof itself didn't seem satisfactory to me. It was me who wrote the proof in the first place, so I will refactor it later.